### ISSUE TITLE
AttributeError: 'FlaskClient' object has no attribute 'cookie_jar'

### ISSUE THREAD

**Issue by LiadOz** [vmalloc/Flask-Loopback#19]
The FlaskClient class inherits from werkzeug Client class, the cookie_jar attribute was removed in the following commit https://github.com/pallets/werkzeug/commit/56f820790a7ca28df972bc50c6790f99cdf5c793
There are two usages of this attribute in flask_loopback:
```python
    def handle_request(self, session, url, request):
        assert url.scheme
        path = "/{0}".format(url.split("/", 3)[-1])
        request_data = request.body
        if hasattr(request_data, 'read'):
            request_data = request_data.read()
        open_kwargs = {
            'method': request.method.upper(), 'headers': iteritems(request.headers), 'data': request_data,
            'environ_base': {'REMOTE_ADDR': _get_hostname()},
            'base_url': '{0.scheme}://{0.netloc}'.format(url),
        }
        with ExitStack() as stack:
            for handler in self._request_context_handlers:
                try:
                    stack.enter_context(handler(request))
                except CustomHTTPResponse as e:
                    return e.response

            self._test_client.cookie_jar.clear()
            for cookie in request._cookies:
                self._test_client.cookie_jar.set_cookie(cookie)
```
The current implementation of the Client provides `get_cookie` `set_cookie` and `delete_cookie` functions. It doesn't look like there is a way to clear the previous cookies. But since only flask_loopback inserts the cookies, it should be able to remove all of the previously inserted cookies.

**Comment by vmalloc** [comment]
Hi @LiadOz! Thank you very much for reporting this issue! Unfortunately I have very little time to maintain Flask-Loopback nowadays so if you feel like crafting a PR to solve this issue (along with a test for it preferably) that would be much appreciated. Thanks!

**Comment by LiadOz** [comment]
I'll look into making a PR, however, the requirements file would need to have `werkzeug >= 3.0.0` dropping support to anything up until 3.7.

**Comment by vmalloc** [comment]
I think this is fine at this point

**Comment by LiadOz** [comment]
Fixed in #20
